packages/common/src/utils/wallet.ts

```diff
@@ -125,6 +126,30 @@ export const convertJSBIToAmountObject = (amount: JSBI, decimals: number) => {
   }
 }
 
+export const convertQuoteStringToAmountObject = (
```

Comment:
it would be nice of AmountObject was something we typed

Reply:
it is! i could return `as AmountObject` or specify the return type in the fn.

Reply:
specify return type is good. it actually is different right? the caller will only know the shape versus knowing that it is AccountObject